docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -455,150 +455,150 @@ The default is `true`.
     You can override the default DNS server if it's unreliable or doesn't serve TCP DNS requests, using the
     [experimental option](#xhelp) `--Xp2p-dns-discovery-server=<HOST>`.
 
-### `engine-rpc-http-port`
+### `engine-host-allowlist`
 
 === "Syntax"
 
     ```bash
-    --engine-rpc-http-port=<PORT>
+    --engine-host-allowlist=<hostname>[,<hostname>...]... or "*"
     ```
 
 === "Example"
 
     ```bash
-    --engine-rpc-http-port=8550
+    --engine-host-allowlist=localhost,127.0.0.1
     ```
 
 === "Environment variable"
 
     ```bash
-    BESU_ENGINE_RPC_HTTP_PORT=8550
+    BESU_ENGINE_HOST_ALLOWLIST=localhost,127.0.0.1
     ```
 
 === "Configuration file"
 
     ```bash
-    engine-rpc-http-port=8550
+    engine-host-allowlist=localhost,127.0.0.1
     ```
 
-The listening port for the Engine API calls (`ENGINE`, `ETH`) for JSON-RPC over HTTP.
+A comma-separated list of hostnames to allow for Engine API access (applies to both HTTP and WebSockets).
 
-### `engine-rpc-ws-port`
+!!!tip
+
+    To allow all hostnames, use `"*"`. We don't recommend allowing all hostnames in production
+    environments.
+
+### `engine-jwt-enabled`
 
 === "Syntax"
 
     ```bash
-    --engine-rpc-ws-port=<PORT>
+    --engine-jwt-enabled
     ```
 
 === "Example"
 
     ```bash
-    --engine-rpc-ws-port=8551
+    --engine-jwt-enabled=true
     ```
 
 === "Environment variable"
 
     ```bash
-    BESU_ENGINE_RPC_WS_PORT=8551
+    BESU_ENGINE_JWT_ENABLED=true
     ```
 
 === "Configuration file"
 
     ```bash
-    engine-rpc-ws-port=8551
+    engine-jwt-enabled=true
     ```
+Enables or disables authentication for Engine APIs. The default is `false`.
 
-The listening port for the Engine API calls (`ENGINE`, `ETH`) for JSON-RPC over WebSockets.
-
-### `engine-host-allowlist`
+### `engine-jwt-secret`
 
 === "Syntax"
 
     ```bash
-    --engine-host-allowlist=<hostname>[,<hostname>...]... or "*"
+    --engine-jwt-secret=<FILE>
     ```
 
 === "Example"
 
     ```bash
-    --engine-host-allowlist=localhost,127.0.0.1
+    --engine-jwt-secret=jwt.hex
     ```
 
 === "Environment variable"
 
     ```bash
-    BESU_ENGINE_HOST_ALLOWLIST=localhost,127.0.0.1
+    BESU_ENGINE_JWT_SECRET="publicKey.pem"
     ```
 
 === "Configuration file"
 
     ```bash
-    engine-host-allowlist=localhost,127.0.0.1
+    engine-jwt-secret="jwt.hex"
     ```
 
-A comma-separated list of hostnames to allow for Engine API access (applies to both HTTP and WebSockets).
-
-!!!tip
-
-    To allow all hostnames, use `"*"`. We don't recommend allowing all hostnames in production
-    environments.
+The shared secret used for JSON-RPC (both HTTP and WebSocket) authentication of Engine API clients.
```

Comment:
"Shared secret used to authenticate consensus clients when using the Engine JSON-RPC API (both HTTP and WebSocket)"?

Is this used for each call to the API, or only when connecting the first time etc?

Also, could we perhaps link to how we create this?

Reply:
Whether it is used each time or only once depends on if you're using traditional RPC or WS. I'm not sure this is relevant to include in the doc, since it doesn't affect usage of the option.